### ISSUE TITLE
The "remote" backend encountered an unexpected error while reading the organization settings: unauthorized.

### ISSUE THREAD

**Issue by jdiazdiv** [hashicorp/terraform#28231]
### Terraform Version
I tried with 0.14.9, 0.14.6 and 0.13.5.
```
❯ tf version
Terraform v0.14.9
```
### Terraform Configuration Files

```terraform
terraform {
  required_providers {
    google = {
      source  = "hashicorp/google"
      version = "3.57.0"
    }
  }
  backend "remote" {
    organization = "divisadero"

    workspaces {
      name = "pruebas-juan_adh"
    }
  }
}
```

### Debug Output

https://gist.github.com/jdiazdiv/4f49f661ae9b0a308f08256b089be165

### Expected Behavior
Correct terraform init.

### Actual Behavior

Cannot innit nor validate or apply.

### Steps to Reproduce

`terraform init`

### Additional Context
It was working as expected until yesterday. It doesn't work in new projects but also in older ones.

**Comment by jakewan** [comment]
I came across this issue while Googling the error shown in the title. At least in my case, the problem turned out to be caused after I had deleted my user API token and created a new one. I had set the `TFE_TOKEN` environment variable to the new API token and thought that was sufficient. However, my `$HOME/.terraform.d/credentials.tfrc.json`file still referenced one of the deleted user API tokens. *After* I updated credentials.tfrc.json with the new API token, I was able to execute `terraform init` successfully.
